nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -477,19 +477,24 @@ public Description matchSwitch(SwitchTree tree, VisitorState state) {
       return Description.NO_MATCH;
     }
 
-    ExpressionTree switchExpression = tree.getExpression();
-    if (switchExpression instanceof ParenthesizedTree) {
-      switchExpression = ((ParenthesizedTree) switchExpression).getExpression();
+    ExpressionTree switchSelectorExpression = tree.getExpression();
```

Comment:
This variable rename is to avoid confusion now that we have switch expressions.  Also, add a comment to explain the stripping of one level of parens